README.md

```diff
@@ -1,7 +1,8 @@
 # nyc
 
-[![Build Status](https://travis-ci.org/istanbuljs/nyc.svg?branch=master)](https://travis-ci.org/istanbuljs/nyc)
-[![Coverage Status](https://coveralls.io/repos/istanbuljs/nyc/badge.svg?branch=)](https://coveralls.io/r/istanbuljs/nyc?branch=master)
+[![Linux Build Status](https://img.shields.io/travis/istanbuljs/nyc/master.svg?label=Linux%20build)](https://travis-ci.org/istanbuljs/nyc)
+[![Windows Build Status](https://img.shields.io/appveyor/ci/istanbuljs/nyc/master.svg?label=Windows%20build)](https://ci.appveyor.com/project/istanbuljs/nyc/branch/master)
+[![Coverage Status](https://img.shields.io/coveralls/github/istanbuljs/nyc/master.svg)](https://coveralls.io/r/istanbuljs/nyc?branch=master)
```

Comment:
Why change the badge URL?

Reply:
Just so that every badge is using shields.io. I can revert this, I just find it more consistent.

Reply:
Hmmm actually leave this, it will improve HTTP connection sharing.

Reply:
Yup, that's my main reason for the switch too.